### PR TITLE
Add initial tutorial and Discourse landing pages

### PR DESCRIPTION
**Pull request by taurus-forever**
K8s copy of VM Tutorial https://github.com/canonical/mysql-operator/pull/98

### CODE REVIEW

**Review by paulomach — APPROVED** [review]

File: docs/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+Before we start, make sure your machine meets the following requirements:
+- Ubuntu 20.04 (Focal) or later.
+- 8GB of RAM.
+- 2 CPU threads.
+- At least 20GB of available storage.
+- Access to the internet for downloading the required snaps and charms.
+
+
+## Prepare Microk8s
+The fastest, simplest way to get started with Charmed MySQL K8s is to set up a local Microk8s lightweight Kubernetes. Microk8s is the best Kubernetes experience for developers, DevOps, cloud and edge; Charmed MySQL K8s will be run in Microk8s Kubernetes and managed by Juju. While this tutorial covers the basics of Microk8s, you can [explore more Microk8s here](https://microk8s.io/). To install Microk8s on Ubuntu 22.04 execute:
```
Comment (jnsgruk) [inline docs/tutorial.md:25]: One recommendation if I may -- I find starting the tutorial with getting Multipass installed and running the tutorial in the fresh VM is a nice way of counteracting any weirdness on the reader's system :)

Comment (jnsgruk) [reply docs/tutorial.md:25]: (It also means at the end of the tutorial they only ever have to `multipass delete <name>` to clean up!)

Comment (taurus-forever) [reply docs/tutorial.md:25]: Tnx @jnsgruk for the feedback!
I wish to use multipass (in fact I am using it locally already, but constantly facing [this issue](https://github.com/canonical/multipass/issues/2772).
Maybe you can rise a priority there due to high visibility in the first copy&paste command?

Comment (taurus-forever) [reply docs/tutorial.md]: Done in https://github.com/canonical/mysql-k8s-operator/pull/148/commits/088207a84cd5ae45fbc7c6db25d8f309503855f1. Resolved.

**Review by deusebio — APPROVED** [review]
The tutorial looks very nice and neat! 

Possibly, but it can be done on a separate ticket, we could also add a small section about mysql router and how to integrate it

**Comment by taurus-forever** [comment]
> a small section about mysql router and how to integrate it

Sure, it is following!

**Pull request merged**